github/resource_github_actions_runner_group.go

```diff
@@ -0,0 +1,232 @@
+			selectedRepositoryIDs = append(selectedRepositoryIDs, int64(id.(int)))
+		}
+	}
+
+	ctx := context.Background()
+
+	log.Printf("[DEBUG] Creating organization runner group: %s (%s)", name, orgName)
+	runnerGroup, resp, err := client.Actions.CreateOrganizationRunnerGroup(ctx,
+		orgName,
+		github.CreateRunnerGroupRequest{
+			Name:                  &name,
+			Visibility:            &visibility,
+			SelectedRepositoryIDs: selectedRepositoryIDs,
+		},
+	)
+	if err != nil {
+		return err
+	}
+	d.SetId(strconv.FormatInt(runnerGroup.GetID(), 10))
+	d.Set("etag", resp.Header.Get("ETag"))
+	d.Set("allows_public_repositories", runnerGroup.GetAllowsPublicRepositories())
+	d.Set("default", runnerGroup.GetDefault())
+	d.Set("id", runnerGroup.GetID())
+	d.Set("inherited", runnerGroup.GetInherited())
+	d.Set("name", runnerGroup.GetName())
+	d.Set("runners_url", runnerGroup.GetRunnersURL())
+	d.Set("selected_repositories_url", runnerGroup.GetSelectedRepositoriesURL())
+	d.Set("visibility", runnerGroup.GetVisibility())
+	d.Set("selected_repository_ids", selectedRepositoryIDs) // Note: runnerGroup has no method to get selected repository IDs
+
+	return resourceGithubActionsRunnerGroupRead(d, meta)
+}
+
+func resourceGithubActionsRunnerGroupRead(d *schema.ResourceData, meta interface{}) error {
+	err := checkOrganization(meta)
+	if err != nil {
+		return err
+	}
+
+	client := meta.(*Owner).v3client
+	orgName := meta.(*Owner).name
+
+	runnerGroupID, err := strconv.ParseInt(d.Id(), 10, 64)
+	if err != nil {
+		return err
+	}
+	ctx := context.WithValue(context.Background(), ctxId, d.Id())
+	if !d.IsNewResource() {
+		ctx = context.WithValue(ctx, ctxEtag, d.Get("etag").(string))
+	}
+
+	log.Printf("[DEBUG] Reading organization runner group: %s (%s)", d.Id(), orgName)
+	runnerGroup, resp, err := client.Actions.GetOrganizationRunnerGroup(ctx, orgName, runnerGroupID)
+	if err != nil {
+		if ghErr, ok := err.(*github.ErrorResponse); ok {
+			if ghErr.Response.StatusCode == http.StatusNotModified {
+				return nil
+			}
+			if ghErr.Response.StatusCode == http.StatusNotFound {
+				log.Printf("[WARN] Removing organization runner group %s/%s from state because it no longer exists in GitHub",
+					orgName, d.Id())
+				d.SetId("")
+				return nil
+			}
+		}
+		return err
+	}
+
+	d.Set("etag", resp.Header.Get("ETag"))
+	d.Set("allows_public_repositories", runnerGroup.GetAllowsPublicRepositories())
+	d.Set("default", runnerGroup.GetDefault())
+	d.Set("id", runnerGroup.GetID())
+	d.Set("inherited", runnerGroup.GetInherited())
+	d.Set("name", runnerGroup.GetName())
+	d.Set("runners_url", runnerGroup.GetRunnersURL())
+	d.Set("selected_repositories_url", runnerGroup.GetSelectedRepositoriesURL())
+	d.Set("visibility", runnerGroup.GetVisibility())
+
+	log.Printf("[DEBUG] Reading organization runner group repositories: %s (%s)", d.Id(), orgName)
+	runnerGroupRepositories, resp, err := client.Actions.ListRepositoryAccessRunnerGroup(ctx, orgName, runnerGroupID)
```

Comment:
```suggestion
	runnerGroupRepositories, resp, err := client.Actions.ListRepositoryAccessRunnerGroup(ctx, orgName, runnerGroupID)
	if err != nil {
		return err
	}
```

Fixes a linter warning:
> github/resource_github_actions_runner_group.go:172:33: ineffectual assignment to `err` (ineffassign)

Reply:
```suggestion
	runnerGroupRepositories, _, err := client.Actions.ListRepositoryAccessRunnerGroup(ctx, orgName, runnerGroupID)
```

Fixes linter warning: 
> github/resource_github_actions_runner_group.go:172:33: ineffectual assignment to `err` (ineffassign)